src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -219,18 +219,18 @@ class PanelMediaBrowser extends LitElement {
       return;
     }
 
-    if (item.media_content_type.startsWith("audio/")) {
+    const resolvedUrl = await resolveMediaSource(
```

Comment:
We now always resolve the URL, because only the resolved URL can tell us correctly what content type it is. Noticed because TTS providers don't tell their mime type until you generate the file.